pkgs/development/interpreters/spidermonkey/common.nix

```diff
@@ -174,12 +145,6 @@ stdenv.mkDerivation (finalAttrs: rec {
 
   enableParallelBuilding = true;
 
-  # cc-rs insists on using -mabi=lp64 (soft-float) for riscv64,
-  # while we have a double-float toolchain
-  env.NIX_CFLAGS_COMPILE = lib.optionalString (
```

Comment:
This is probably triggering the mass rebuild but not going to bother, staging it is.